.github/workflows/check-generated-files.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           diff --unified "$old" "$new"
 
       - name: generate requirements-dev.txt
-        run: poetry export --extras all --with dev --with test --with docs --without-hashes --no-ansi > requirements-dev.txt
+        run: poetry export --extras all --with dev --with test --with docs --with quarto --without-hashes --no-ansi > requirements-dev.txt
```

Comment:
Once we go live with the new docs site I will:

1. Remove the mkdocs (and related plugins) dependencies from the `docs` group
2. Move the `quarto` group's dependencies to the `docs` group
3. Remove the `quarto` group